src/components/what-we-do/history.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+const History = ({ title, historyMilestones }) => {
+  return (
+    <div>
```

Comment:
Just a little thing, you can use a react fragment instead of a div if you don't need the div explicitly. You can just replace the div with `<> </>`